base_dav/views/dav_collection.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <record id="view_dav_collection_tree" model="ir.ui.view">
+        <field name="model">dav.collection</field>
+        <field name="arch" type="xml">
+            <tree>
+                <field name="name"/>
+                <field name="dav_type"/>
+                <field name="model_id"/>
+                <field name="domain"/>
+            </tree>
+        </field>
+    </record>
+    <record id="view_dav_collection_form" model="ir.ui.view">
+        <field name="model">dav.collection</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <field name="name"/>
+                    <field name="dav_type"/>
+                    <field name="url" widget="url" attrs="{'invisible': [('url', '=', False)]}"/>
+                </group>
+                <group string="Access">
+                    <field name="model_id"/>
+                    <!-- TODO: use widget="domain" when we can set the model dynamically /-->
+                    <field name="domain"/>
+                    <field name="field_uuid" domain="[('model_id', '=', model_id)]"/>
+                    <field name="rights"/>
+                </group>
+                <group string="Additional field mapping">
+                    <field name="field_mapping_ids" nolabel="1"/>
```

Comment:
To set the default value to link correctly a field mapping from the UI:

```suggestion
                    <field name="field_mapping_ids" nolabel="1" context="{'default_collection_id': active_id}/>
```

Reply:
(BTW I did this change by hand on http://3432983-61-2e5e17.runbot2.odoo-community.org/web?debug=1#id=2&view_type=form&model=dav.collection&action=116 )

It fixes it.